src/server/req_runjob.c

```diff
@@ -1437,12 +1398,11 @@ post_sendmom(struct work_task *pwt)
 						NULL, NULL, log_buffer);
 				}
 			}
-			else {
-				/* if the job is a normal job */
-				job_attr_def[(int) JOB_ATR_Comment].at_decode(
-						&jobp->ji_wattr[(int) JOB_ATR_Comment], NULL,
-						NULL, log_buffer);
-			}
+
+			/* if the job is a normal job */
+			job_attr_def[(int) JOB_ATR_Comment].at_decode(
+					&jobp->ji_wattr[(int) JOB_ATR_Comment], NULL,
+					NULL, log_buffer);
```

Comment:
Same as the above comment.

Reply:
fixed in rc 23 https://github.com/PBSPro/pbspro/pull/590/commits/75a2cfae3e21fc17658ef792e406c158ecdc6314